volcli/commands.go

```diff
@@ -34,6 +34,7 @@ var Commands = []cli.Command{
 		Subcommands: []cli.Command{
 			{
 				Name:        "upload",
+				Flags:       VolmasterFlags,
```

Comment:
@unclejack I think `VolmasterFlags` is now better put in `volcli.GlobalFlags`, that way we don't have to repeat it for every sub-command. wdyt?